src/components/OcBreadcrumb.vue

```diff
@@ -69,6 +77,14 @@ export default {
       type: [Boolean],
       default: false,
     },
+    /**
+     * Aria-label for the home icon
+     */
+    homeAccessibilityLabel: {
```

Comment:
```suggestion
    homeAccessibleLabel: {
```
Please rename - we have that naming (`accessibleLabel`) in other locations already and I would like to keep it consistent.